server/src/com/cloud/storage/VolumeApiServiceImpl.java

```diff
@@ -1137,6 +1137,14 @@ private VolumeVO orchestrateResizeVolume(long volumeId, long currentSize, long n
         UserVmVO userVm = _userVmDao.findById(volume.getInstanceId());
         StoragePoolVO storagePool = _storagePoolDao.findById(volume.getPoolId());
         boolean isManaged = storagePool.isManaged();
+
+        List<Volume> volumes = new ArrayList<Volume>();
```

Comment:
Are you using this variable?

Reply:
don't think so and also these methods here are to huge for my puny brain

Reply:
My brain only works with up to 10 lines of code methods. Anything else is just a blur.

I think the code is ok; we just need to check if this variable is used; if it is not, please remove it.

Reply:
Removed, unused code.